src/Compilers/CSharp/Portable/FlowAnalysis/CSharpDataFlowAnalysis.cs

```diff
@@ -101,6 +101,30 @@ public override ImmutableArray<ISymbol> DataFlowsIn
             }
         }
 
+        /// <summary>
+        /// The set of local variables which are definitely assigned a value when a region
+        /// is entered.
+        /// 
+        /// The set of variables that are definitely assigned a value when a region is exited is
+        /// the union of <see cref="DefinitelyAssignedOnEntry"/> and <see cref="AlwaysAssigned"/>.
+        /// </summary>
+        public override ImmutableArray<ISymbol> DefinitelyAssignedOnEntry
+        {
+            get
+            {
+                if (_definitelyAssignedOnEntry.IsDefault)
+                {
+                    _succeeded = !_context.Failed;
+                    var result = _context.Failed
```

Comment:
No, this is only done as part of data flows in.

Reply:
k

Reply:
#Resolved.